resources/git-hooks/commit-msg

```diff
@@ -106,11 +109,18 @@ function print_error() {
   echo -e "Valid scopes: ${t_grn}${scopes[@]}${t_end}"
   echo -e "Max length (first line): ${t_grn}$max_length${t_end}"
   echo -e "Min length (first line): ${t_grn}$min_length${t_end}"
-  echo -e "Optionally, append ${t_grn}Fixes #1234${t_end}\n"
-  echo -e "${t_cyn}Example:${t_end} fix(windows): Re-attaches the widget plug which had fallen out. Fixes #1111"
+  echo -e "If possible, append git trailers:"
+  echo -e " * ${t_grn}Fixes: #1234${t_end}"
+  echo -e " * ${t_grn}Fixes: KEYMAN-MODULE-XYZ${t_end}"
+  echo -e " * ${t_grn}Cherry-pick-of: #2468${t_end}"
+  echo -e " * ${t_grn}Co-authored-by: Firstname Lastname <email@keyman.com>${t_end}"
+  echo -e "${t_cyn}Example:${t_end} fix(windows): Re-attach the widget plug which had fallen out"
   echo -e "${t_cyn}Reference${t_end}: https://github.com/keymanapp/keyman/wiki/Pull-Request-and-Commit-workflow-notes"
   echo -e ""
-  echo -e "Tips: Don't include a period at the end of the first line; best to put 'Fixes #1111' on a line of its own."
+  echo -e "Tips: "
+  echo -e " * Don't include a period at the end of the title"
+  echo -e " * Always include a blank line before trailers"
+  echo -e " * Use imperative, present tense ('attach' not 'attaches', 'attached' etc)"
```

Comment:
```suggestion
  echo -e " * Use imperative, present tense ('attach' instead of 'attaches', 'attached' etc)"
```

Reply:
Should we mention https://github.com/keymanapp/keyman/wiki/Pull-Request-and-Commit-workflow-notes for details? (Which, by the way, needs to be updated to include trailers...)

Reply:
The reference is there on line 118